nca/Parsers/IngressPolicyYamlParser.py

```diff
@@ -155,9 +155,9 @@ def segregate_longest_paths_and_make_dfa(parsed_paths):
                 path_regex = path_string
             else:
                 if path_string:
-                    path_regex = path_string + '|' + path_string + '/' + allowed_chars + '*'
+                    path_regex = path_string + allowed_chars + '*'
```

Comment:
I'm not sure this is correct. `/foo` should not match `/foobar`. It should only match paths like `/foo/bar`. See [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types)

Reply:
Right, thanks. Added the '/' separator back.